doc/tutorials/spike_frequency_adaptation/nestml_spike_frequency_adaptation_tutorial.ipynb

```diff
@@ -0,0 +1,1189 @@
6cmQP/ZQQsewRb4dziIfDHUSAd47xQuXlmXUeOY/8gShHIcQq0TzNJ/157IeU38pSjhvqkrZBeR+amC/jWasBpkc8ZjzqimIuWHNWW15CZZmE1eSDCJrOk0xq2nutaooyr04jXjn3kPPIH4hyFEK8KkE9WX6C3xe63eOVpjzUhk95sMIG6yujEzMu+TPZM1lYpsxX5ow7lyJo5sx4a/po/zAJM1ZW5jV3xppfrXWqh5R4NnJHziN/I8pRCPHbYRtEQcoac015CVVlwQgFHJuJPWNBuiduISFi2WM9P9Pl+ckbCf9ynrCHRXtJz1CcoXgSkLl1g/SoDplfrxDlKIR4fdiO19DMjxtpVK25qc8X0c13LC+hJMSPjyS7O4e9KuZgLEHPYByQucwXySN0j/GiKuSZdYa0Z9fcW+sqSqgojUZUhx8R5ShkJGyx1n7orhxLJDkSwH5BXiuYTjNu01M53GQuciBs68MrRNB0B5lX95C5dRc5j/yBKEchIy3WusCCy1hemA5TUQP/L3a7/tAe4g1Kay0bsA2Zi/EZq2BJ/3Cc3iHxdmTLWE5qCf/KE2lV4Cpjnekyt84g+Zz+RpSjkJG+cXlUZS0ELviwbFBjbcB9wwkGYgkg+J8vG6IaQuk07T3BMXj4nbBVxfQL1rxWlBZRI6GyjiIhi+4ych7JfuAlohyFDL81L7U6aYM/xjOaseTlgeEEPSG2jMvhlu4llBCxyUmPiff3mvY7Yz17IHOZL2l5MTaDh9/aRwSRtGfWnNviIsWUqvBVcvWEMQzKch55iyhHIcPLzvVjheAESRC3EqXbe4Mz5swZJ1k54gKZJMRnT5DWtN+xz+W0iPcbcxIRMN0jVamypowiH/RRDBN9Q3H6h6MX1eFHRDkKGX4QfMetGhSQxe4375vTBPGeuIXMxcRMlnNQaANMkBnLg2FZ4adWl1FaLMdxtkzWxFvWdH5MZPCUuc2PSY3JMr+eIrtxyLAWV5VPyhJb46kNUFnKMG1QYyaB90QzLGrMw0gS4rPGWh9KGUK9kDvi4XAHySN0D3lm3UPOI/8gylHIaDUFXy8W1phWZp8fUhONGfw77nyQmPGREEpJiM+MsfI5plWXUSLejqwZq2dMGPeZQmNN61A8QWd/DJA17RTj5XMJzmBNr4Qs+4fQnWxKqUeUUnqcr9gY11+ilHpOKTWolNqnlPqCUsp7l0uOWIvLy3CXMQ//ACz00RtUmCzjY92TadVlFEc8ZjxMXsJCMZJzIPOVL6IcOU+Q2kcEjURS0yHKkWvIeeQfAqsETMD/A3466rVq4IfA/fYXlVKvB+4GHgE+DJwA3AA0Ate5PE5X8JtVJ4iHvzXmqVXhzAMI4j1xC0mIzx55fpxBa+27/ToMiIDpHkf6hjHbKAbC4Bk0rGe3KESG2aASOuVIa/3A6NeUUleZ//zVqB99A9gEvFZrHTev7QY+q5T6rtZ6q6uDdQG/eWr8LkiNmSTt8zFnQxBDHV1jgrmQhPixmaggQ+SenzwZPZXdg3GG40nAP/t10JAmpS4zan590UcxJEz07E6rKY98VIfXREUauALoA+6xXlBKHQscC/zYUoxMfoCxJVxe0BE6wFA8wZB52DbWFX7jGr3Y+4bi9AWwLGXYlQe/KdBeInORHeLtcA4R4t0h7DmjXiJz6y6pvVXOI88JnedoNEqpGcAFwG+01n22H51ifl9nv15rfUgpdcD287H+ZiMwY9TLSx0Ybl6UlxSz7Yuvo3swPqZVolA8t+8oJ3z+vrQ8lyAs9lP/5wFQ0Gs1gA3AmDOlo2+YEz5/H4D09QFe953HUArpKZEhO9t6jedHQyxhLOwwrY9C8pcXmnl4WyuJ5MgGKc9f/nzpT1v42t+2pgyEYPTiEfLn54/v4VdP7yWWGJlbeWad47xvPopShkEZZG79QOiVI+BfMD7n6JC62eb35jF+pxmYM8HfvB747/yH5jxKKeorSz1570bTzZ7UIwK4xdLGGi+GNCn2TahnKBhjzoZGW+jDy+7JjOB/vmywFynpHX2vZ1QXejiBwHp+dIDWtF+ZUVvO7vY+4kmdNpdFChZNl+cvF2rKS6gqK6Z/2IiasCtG86dWUl4SjPYRfqWxtpwDRwcYTiQZtilG1WXFzPIgOiVMTHweyd7qNb5WjpRSRUCmpp8hre2+ihRXAG3A6FykSuv3xvidQaBugvf6AXDnqNeWYgvbiyKffN1K5k+pfJmScczsOk6Z3+DNoCbhkpPm0DMY58DR/rTXZ9SWc/WZCz0alXO88+xFlBSrtBAegHlTqrj4pIn0//DxquUz+H9vPp6drb1pr9dXlnJVCO61G3zo3GVMqSqjc2A47fUlM2o4Z/lo57kwEf/7lhP47boDDMUTaa+fsXgacxsqx/ktYSIqSov56TvX8OCW1rTm4yVFiktOmuvhyMLBTVeu5t6NB4nbvJwKxfnHNlJZJopnPpyzYgZfvPR4dreln0cNlWVcfZacR16jxtYn/IFSai3wcIaXHzO6gIJSagmwC/ie1vrDo372ceDrwAKt9f5RP3sGSGitz8pirMcBL7744oscd9xxmf6aIAiCIAiCIAgusHnzZo4//niA47XWmzP5HV97joCtwLszvHas8LgrzO+jQ+rs188G9o/62WzgmQzfVxAEQRAEQRCEEOBr5Uhr3QLcnMefuALYpbV+aoyfbTC/r8GmCCml5gDzgB/n8b6CIAiCIAiCIASM0JbyVkqdAhwD3D7Wz03X2lbg/Uope/DsdYAG7nJ9kIIgCIIgCIIg+AZfe47y5Erz+1ghdRafAO4F7ldK/Ro4HvgQ8FOt9Usuj08QBEEQBEEQBB8RSs+RWeXu7cBzWutt412ntf4T8BZgKnCj+e8vAx8sxDgFQRAEQRAEQfAPofQcaa2TGHlDmVx7N3C3m+MRBEEQBEEQBMH/hNJzJAiCIAiCIAiCkC2iHAmCIAiCIAiCICDKkSAIgiAIgiAIAhDSnCOPKAPYuXOn1+M
```

Comment:
Could you add a line or two about the significance of the CV comparison plot?

Reply:
I've turned this into a task for the participant.